File: kotlin/test/programmers/hash/finishrun/SolutionTest.java
package programmers.hash.finishrun;

import org.junit.Test;

import static org.junit.Assert.*;

public class SolutionTest {
    @Test
    public void testFinishRun() {
        Solution solution = new Solution();
        String[] participants = {"leo", "kiki", "eden"};
        String[] completion = {"kiki", "eden"};
        assertEquals("leo", solution.solution(participants, completion));
    }
}

File: test/q1182/SolutionTest.kt
package q1182

import org.junit.Assert.*
import org.junit.Test

class SolutionTest {
    private val solution = Solution()

    @Test
    fun example1() {
        assertEquals(1, solution.solution(5, 0, intArrayOf(-7, -3, -2, 5, 8)))
    }

    @Test
    fun only1() {
        assertEquals(1, solution.solution(5, 5, intArrayOf(-10, -11, -13, -14, 5)))
    }
}